Src/CSharpier/SyntaxPrinter/ArgumentListLikeSyntax.cs

```diff
@@ -11,19 +13,23 @@ public static class ArgumentListLike
             SyntaxToken openParenToken,
             SeparatedSyntaxList<ArgumentSyntax> arguments,
             SyntaxToken closeParenToken
-        ) =>
-            Doc.Concat(
-                Token.Print(openParenToken),
-                arguments.Any()
-                    ? Doc.Concat(
-                          Doc.Indent(
-                              Doc.SoftLine,
-                              SeparatedSyntaxList.Print(arguments, Argument.Print, Doc.Line)
-                          ),
-                          Doc.SoftLine
-                      )
-                    : Doc.Null,
-                Token.Print(closeParenToken)
-            );
+        ) {
+            var docs = new List<Doc> { Token.Print(openParenToken) };
+
+            if (arguments.Any())
+            {
+                docs.Add(
```

Comment:
Why is this change needed?
I'm not saying I prefer the old code over this, but I would like to understand why you're doing this.

Is there some penalty for having `Doc.Null`? If yes, wouldn't it be easier to filter them out in `Doc.Concat()`, causing them to be removed everywhere?